active/gridsyntax/GridSyntaxSpec.md

```diff
@@ -0,0 +1,335 @@
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+## Overview
+A new XAML language feature that allows the initialization of collection-type properties (even if they are read-only) using element attribute syntax.
+
+## Implementation Details
+Read-only collection-type properties will be assigned a new CreateFromString attribute. The CreateFromString attribute will allow them to be defined/created from a comma-delimited string. In terms of Grid, this means that the ColumnDefinitions and RowDefinitions properties are able to be defined by providing a comma-delimited string of ColumnDefinitions (Width values) and a comma-delimited string of RowDefinitions (Height values), respectively. However, this feature is also applicable to other scenarios within the XAML language where collection-typed read-only properties are created (see Remarks for examples of such scenarios). 
+
+In order to support this language feature in Grid, a mechanism is needed that allows ColumnDefinition and RowDefinition objects to be createable from their Width/Height properties, respectively. To achieve this, the content property of ColumnDefinition is set to the Width property, and the content property of RowDefinition is set to the Height property. Developers can assign a comma-delimited string of integers to the ColumnDefinitions property, and each of those integers is parsed as a Width values, creating a ColumnDefinition (and the same applies to RowDefinition with Height). This will be used to make the new syntax (see below) fully functional.
+
+
+# Examples
+### New succinct syntax
+The code below has the same functionality as the code shown above with the original syntax. It creates a grid and defines five different rows and columns, each with their own specific height/width, and adds them to the Grid. 
+```xml
+<Grid ColumnDefinitions="1*, 2*, Auto, *, 300"
+      RowDefinitions="1*, Auto, 25, 14, 20">
+</Grid>
+```
+
+### New succinct syntax for more complex types
+The code below shows an example of how the syntax would be used for collection-typed properties that have more complex value types.
+
+This shows how Point objects will be delimited by a comma, but indvidual values will be encased in single quotes.
+```xml
+<Foo PointList=" '1,2', '9,4', '2,6'"/>
+```
+
+This shows how String values will be supported, even if they include commas or single quotes inside of them. Each string will be encased in single quotes and delimited by commas, and single quotes inside of a string will be escaped by using an HTML entity. The collection shown below has 4 elements, and 'hello, world!' is one element.
```

Comment:
Following the XML grammar, if the attribute value is surrounded by `'`, then `"` is the grouping character that is used within the string. Basically, attribute values can be enclosed with either `'` or `"`, which then allows the other quotation mark to be used within the value itself without needing to be escaped with an XML character entity.